modules/api/current.js

```diff
@@ -142,16 +144,23 @@ module.exports = {
         attributes: utils.createAttributes(feature.tags),
         codeRef: featureCodeRef,
       };
+
+      if (isFirstFeatureInLaunch) {
+        this.storage.setCurrentFeatureUri(pickleFeatureUri);
```

Comment:
Looks like if we `isNeedToStartFeature`, the `this.storage.setCurrentFeatureUri` method will be called anyway, so this block can be omitted.

Reply:
Yes, u are absolutely right.

But I use this only for one purpose, to clarify everything.
If I delete that block, when I read such code, the first question will be: "Okay, What will happen if it is the first feature in launch?".

`  if (isFirstFeatureInLaunch) {
        this.storage.setCurrentFeatureUri(pickleFeatureUri)} 
        else {
        const previousFeatureTempId = this.storage.getFeatureTempId();
        this.reportportal.finishTestItem(previousFeatureTempId, {
          endTime: this.reportportal.helpers.now(),
        });
        this.storage.setCurrentFeatureUri(pickleFeatureUri);
      }`
Such condition gives a more clear understanding in the case if it is the first feature in a launch.

What do u think?